build.sh

```diff
@@ -115,7 +120,13 @@ then
     dotnet build "${root_path}"/Compilers.sln ${build_args} "/bl:${binaries_path}/Build.binlog"
 fi
 
+if [[ "${stop_vbcscompiler}" == true ]]
+then
+    echo "Stopping VBCSCompiler"
+    dotnet "${bootstrap_path}"/bincore/VBCSCompiler.dll -shutdown
```

Comment:
Shouldn't the path being used here be different depending on whether `${use_bootstrap}` is set?

Reply:
Yeah... although finding `VBCSCompiler.dll` is pretty difficult when `${use_bootstrap}` is false. Because stop_vbcscompiler is a pretty niche feature, I'm tempted to just make it be an error to `(${use_bootstrap} == false && ${stop_vbcscompiler} == true)`

Reply:
Hmm. Hadn't thought about that. It actually needs to be first class supported because it's critical to having a sane infrastructure. All the cases we stop the server in our infra are cases where others may need to as well.

Lets work with the CLI team separately from this PR to figure that out.

Reply:
It's moments like these where I really wish the architecture of the server was per-msbuild-invocation and had a strict parent/child relationship and msbuild was responsible for cleaning up the server... oh well, I can see the argument for the perf bonus on multiple repeated invocations on small solutions.